src/js/InfographicTemplates/ADHDTemplateOne.js

```diff
@@ -0,0 +1,372 @@
+        this._textHandler.AddTextElem({
+            textElem: underTitleDiv, 
+            group: header, 
+            x: 50, 
+            y: 100
+        });
+        this._textHandler.SetCSSInfo({
+            id: this._textHandler.GetCurrID(),
+            fontFamily: roboto700,
+            fontSize: '40px',
+            textColor: underTitleDiv.style.color,
+            lineHeight: '1.2',
+            align: 'center',
+        });
+
+        let strip = new RectangleHeader(
+            {
+                x: 1,
+                y: 200,
+                width: this._chartWidth - 2,
+                height: 25,
+                fill: '#ffd34e',
+                group: descBackgroundGroup
+            }
+        );
+        this._graphicsHandler.AddGraphic({
+            type: 'header',
+            graphic: strip,
+            group: descBackgroundGroup
+        });
+    }
+
+    _CreateSectionHeader(){
+        let sectionHead = this._CreateSwitchableContainer({
+            x: 0,
+            y: 250
+        }, 'sectionHead')
+
+        this._main.add(sectionHead)
+
+        var roboto700 = this._quillMap('Roboto', 700);
+
+        let sideBarGroup = new Konva.Group();
+        sectionHead.add(sideBarGroup)
+        let sideBarLeft = new RectangleHeader(
+            {
+                x: 1,
+                y: 0,
+                width: 40,
+                height: 110,
+                fill: '#db9d36',
+                group: sideBarGroup
+            }
+        );
+        this._graphicsHandler.AddGraphic({
+            type: 'header',
+            graphic: sideBarLeft,
+            group: sideBarGroup
+        });
+       
+        let sideBarRightGroup = new Konva.Group();
+        sectionHead.add(sideBarRightGroup)
+        let sideBarRight = new RectangleHeader(
+            {
+                x: 410,
+                y: 0,
+                width: 800-330,
+                height: 110,
+                fill: '#db9d36',
+                group: sideBarRightGroup
+            }
+        );
+        this._graphicsHandler.AddGraphic({
+            type: 'header',
+            graphic: sideBarRight,
+            group: sideBarRightGroup
+        });
+
+        var sectionTitleDiv = document.createElement('div')
+        var sectionTitle = '<p><span style="font-size: 55px; font-family: Roboto, sans-serif; font-weight: 800">'
+            + 'ADD / ADHD</span></p>';
+        sectionTitleDiv.innerHTML = sectionTitle;
+        sectionTitleDiv.style.color = '#145860';
+        this._textHandler.AddTextElem({
+            textElem: sectionTitleDiv, 
+            group: sectionHead, 
+            x: 50, 
+            y: 0
+        });
+        this._textHandler.SetCSSInfo({
+            id: this._textHandler.GetCurrID(),
+            fontFamily: roboto700,
+            fontSize: '55px',
+            textColor: sectionTitleDiv.style.color,
+            lineHeight: '1.2',
+            align: 'center',
```

Comment:
lineHeight and center attributes defined here but not in the CSS for sectionTitle